packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -615,7 +623,8 @@ export async function getOrganization(req: AuthRequest, res: Response) {
   userInfo.forEach(({ id, email, verified, name, _id }) => {
     const memberInfo = members.find((m) => m.id === id);
     if (!memberInfo) return;
-    expandedMembers.push({
+
+    return expandedMembers.push({
```

Comment:
@jdorn Good call. I think in this case, it might actually be better to have a `forOf` loop anyways. `forEach` loops don't support early returns like we're doing on line 625. 

```ts
const expandedMembers: ExpandedMember[] = [];
  for (const user of userInfo) {
    const memberInfo = members.find((m) => m.id === user.id);
    if (!memberInfo) return;

    expandedMembers.push({
      email: user.email,
      verified: user.verified,
      name: name || "",
      ...memberInfo,
      dateCreated: memberInfo.dateCreated || user._id.getTimestamp(),
    });
  }
```

Reply:
That won't work.  If you do a `for` loop, that return there will return the entire outer function, not the loop iteration.

I would revert this whole section to what it was before.